mlir/test/Dialect/Vector/vector-multi-reduction-lowering.mlir

```diff
@@ -298,6 +298,30 @@ func.func @scalable_dim_1d(%A: vector<[4]xf32>, %B: f32, %C: vector<[4]xi1>) ->
 // CHECK:          %[[VAL_4:.*]] = vector.extract %[[VAL_3]][0] : f32 from vector<1xf32>
 // CHECK:          return %[[VAL_4]] : f32
 
+func.func @scalable_dim_2d(%A: vector<2x[4]xf32>, %B: vector<2xf32>, %C: vector<2x[4]xi1>) -> vector<2xf32> {
```

Comment:
Could you differentiate this (and `@scalable_dim_1d`) from `@scalable_dims`? IIUC, the difference is the scalability of the reduction dimension?

Reply:
renamed them to:
vector_multi_reduction_non_scalable_dim
vector_multi_reduction_scalable_dim_1d
vector_multi_reduction_scalable_dim_2d

Reply:
Thanks! This would be a bit clearer:
vector_multi_reduction_non_scalable_reduction_dim
vector_multi_reduction_scalable_reduction_dim_1d
vector_multi_reduction_scalable_reduction_dim_2d

Accepting as is, no need to bikeshed.